blockchain/chaingen/README.md

```diff
@@ -25,8 +25,7 @@ functions.
 
 ## Examples
 
-* [Basic Usage Example]
-  (https://pkg.go.dev/github.com/decred/dcrd/blockchain/v3/chaingen#example-package-BasicUsage)
+* [Basic Usage Example](https://pkg.go.dev/github.com/decred/dcrd/blockchain/v3/chaingen#example-package-BasicUsage)
```

Comment:
Please add 2 spaces at the end of the line so the rendered document properly moves the description to the next line.